src/main/java/com/cleancommunity/user/Volunteer.java

```diff
@@ -66,6 +71,15 @@ public boolean completePosting(Posting post){
         list.add(projectsPosted);
         return list;
     }
+    public int getProjectsCompleted(){
+        return projectsCompleted;
+    }
+    public int getProjectsInProgress(){
+        return projectsInProgress;
```

Comment:
This data will not persist if another user is created by request before user was updated. Should just infer this value via DB query.